include/mptcpd/types.h

```diff
@@ -162,6 +163,10 @@ typedef void (*mptcpd_pm_get_limits_cb)(
         size_t len,
         void *callback_data);
 
+typedef void (*mptcpd_parse_func_t) (
+        struct l_settings *const settings,
```

Comment:
The `const` to the right of the pointer doesn't really provide a benefit since the _pointer_ is passed by value.  Modifying the pointer argument inside the parse function won't impact the original value of the pointer.  Did you intend to make the `l_settings` object itself `const`, i.e.  `struct l_settings const *settings`?

Reply:
>  Did you intend to make the l_settings object itself const, i.e. struct l_settings const *settings?

Yes, I never get that right.